src/WpfMath/Converters/SVGConverter.cs

```diff
@@ -1,49 +1,50 @@
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.Globalization;
-using System.Text;
 using System.Windows.Media;
 
 namespace WpfMath.Converters;
 
 public class SVGConverter
 {
-    private int m_nestedLevel = 0;
+    //private int m_nestedLevel = 0;
```

Comment:
Meh, it was never used, let's just drop it.

Reply:
Perfect, I just dropped it :) now let's just wait for the builds to finish :)